Pack.tests.template.ps1

```diff
@@ -47,7 +46,7 @@ foreach ($packFile in $packFiles) {
         Context 'details.json' {
             
             It "details.json is valid json" {
-                { $contents | ConvertFrom-Json } | Should Not Throw 
+                {$script:json = $contents | ConvertFrom-Json } | Should Not Throw 
```

Comment:
This makes the tests order dependant (other tests will fail if this one isn't run first).  The original call probably needs a try/catch wrapping it.